crmprtd/ec/__init__.py

```diff
@@ -9,6 +9,7 @@
 from sqlalchemy.sql import func
 from sqlalchemy.exc import IntegrityError
 from crmprtd import Timer
+from crmprtd.align import closest_stns_within_threshold
```

Comment:
This only gets used in code that you've obsoleted in this PR. I'd remove this line and the obsolete code.